src/components/CreateWatch.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+import { getActiveFilterKeys } from '../data/filtering'
+import { getFiltersForWatch, getWatchableKeys, create } from '../data/watch'
+import { createdWatch } from '../data/analytics'
+import FilterTag from './FilterTag.svelte'
+import jquery from 'jquery' // $ is already a reserved token in Svelte (stores)
+import { onMount } from 'svelte'
+
+export let filter
+
+let name = ''
+let submitted = false
+
+function onSubmit() {
+  create(name, filter)
+  createdWatch()
+  submitted = true
+}
+
+onMount(() => {
+    jquery('#thisModal').on('hidden.bs.modal', function (e) {
+        name = ''
+        submitted = false
+    })
+});
+
+$: canWatch = getWatchableKeys(getActiveFilterKeys(filter)).length
+$: watchFilters = getFiltersForWatch(filter)
+
+</script>
+{#if canWatch}
+  <button class={$$props.class} data-toggle="modal" data-target="#thisModal">Create Alert</button>
+
+  <div class="modal fade" id="thisModal" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered" role="document">
+      <div class="modal-content">
+        <div class="modal-header bg-primary">
+            <h4 class="modal-title text-white">Create an Alert</h4>
+            <button type="button" class="close text-white" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true" class="align-text-top">×</span>
+            </button>
+        </div>
+        {#if !submitted}
+          <form on:submit|preventDefault={onSubmit}>
+            <div class="modal-body">
+              <div>
+                Alert Settings:
+              </div>
+                {#each watchFilters as f}
+                  <FilterTag label="{f.label}" showCloseButton="{false}" />
+                {/each}
```

Comment:
Sure. This is a "switching gears" thing because single-letter variables are very common in Go. I think the philosophy is that local variables don't always need the clarity that comes from longer names. But consistency with project style trumps that.